app/assets/javascripts/details/shell/action_bar_bridge.js

```diff
@@ -0,0 +1,116 @@
+var ActionBarBridge;
```

Comment:
@dogweather This is the naming convention we are using. Its consistent with the `coffeescript` output for creating constructors.